docs/source/markdown/podman-pull.1.md

```diff
@@ -43,7 +43,7 @@ $ podman pull oci-archive:/tmp/myimage
 ```
 
 ## OPTIONS
-#### **--all-tags**, **a**
+#### **--all-tags**, **-a**
```

Comment:
./bin/podman pull --help | grep -- --all
      --all-tags                All tagged images in the repository will be pulled

--all-tags does not have an alias of -a, and should not.  Just remove the `**a**`